### DISCUSSION TITLE
Recovery Account

### DISCUSSION THREAD

**Discussion by fauzanhlmy** [community/community#146892] [Other Feature Feedback, Questions, & Ideas] [answered]
### Recovery Account

Question

Hi @quoc10973 and @queenofcorgis thanks for reply my latest thread [here](https://github.com/orgs/community/discussions/146122)
unfortunately, I'll already follow all the suggestions and also sending email to support@github.com.
![Screenshot 2024-12-12 115509](https://github.com/user-attachments/assets/3cf95c2c-aa27-4cde-9176-2bcd6ebda464)

but as you can see.. seems like support@github.com blocked an email. could you help what should i do for this situations ? surely i really need my account  back..

**Comment by queenofcorgis** [comment, marked as answer]
Hi there, 

That is expected behavior as that email address does not accept incoming messages. 

For ways to contact support, please review [in our docs](https://docs.github.com/en/support/contacting-github-support) - including [how to create a support ticket](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket). 

Please note the following : 

- The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
- Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information. Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
